6.3/main.py

```diff
@@ -0,0 +1,25 @@
+
+#get a sentence
+#return a list of the length of each word in the sentence
+#using list comprehension
```

Comment:
Write documentation in the right format - 
python
def count_hi_v4(word: str) -> int:
    """
    The function counts how many times the string "hi" appears in a given string.
    @param word: The given string.
    @return: The amount of times the word "hi" appeared.
    """
    return word.count("hi")

Do this for all functions.